test/errors.js

```diff
@@ -1,47 +1,34 @@
 import test from 'ava'
 
-import tmp from './helpers/get-tmp.js'
-import run from './helpers/run-cli.js'
+import tmp from './helpers/tmp.js'
+import cli from './helpers/cli.js'
 
-test('when no output option', (t) => {
-  return run([ 'test/fixtures/a-red.css' ])
-    .then(({ error, code }) => {
-      t.is(code, 1, 'expected non-zero error code')
-      t.regex(error, /No Output specified, either --output, --dir, or --replace option must be passed/)
-    })
+test('multiple input files && --output is set', (t) => {
+  return cli(
+    [ 'test/fixtures/*.css', '-o', tmp() ]
+  )
+  .then(({ error, code }) => {
+    t.is(code, 1, 'expected non-zero error code')
+    t.regex(error, /Must use --dir or --replace/)
+  })
 })
 
-test('when multiple input files and --output is set', t => {
-  return run([
-    'test/fixtures/*.css',
-    '-o', tmp()
-  ])
-    .then(({ error, code }) => {
-      t.is(code, 1, 'expected non-zero error code')
-      t.regex(error, /Must use --dir or --replace/)
-    })
-})
-
-test('when invalid --config option', (t) => {
-  return run([
-    'test/fixtures/*.css',
-    '--config', 'test/fixtures/postcss.config.js',
-    '-o', tmp()
-  ])
-    .then(({ error, code }) => {
-      t.is(code, 1, 'expected non-zero error code')
-      t.regex(error, /ENOENT: no such file or directory/)
-    })
+test('invalid --config option', (t) => {
+  return cli(
+    [ 'test/fixtures/*.css', '-c', 'test/postcss.config.js', '-o', tmp() ]
```

Comment:
This should be

    [ 'test/fixtures/*.css', '-c', 'test/postcss.config.js', '-d', tmp() ]

or 

    [ 'test/fixtures/a.css', '-c', 'test/postcss.config.js', '-o', tmp() ]